### DISCUSSION TITLE
PAC package deploy failure - "Could not find a part of the path ..."

### DISCUSSION THREAD

**Discussion by JPNYPZSW** [microsoft/powerplatform-build-tools#990] [Q&A]
When I use pac package deploy, I'm getting the below error.

`Could not find a part of the path 'C:\Users\xxxxxxxxxx\AppData\Local\Temp\uqoglyqw.cpi\87\tempzips\applicationfoundation-develop\applicationfoundation-develop\ApplicationFoundation\AxLabelFile\LabelResources\ar-AE\FieldDescriptions_OrganizationAdministration_ApplicationFoundation.ar-AE.label.txt'`

I guess the default temp path of package deploy may be too long. Is it possible to change the temp path?

**Comment by petrochuk** [comment]
In most cases path length should not be a problem. Pac cli has Windows long path support enabled [Maximum Path Length Limitation](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry)
